src/runtime_math.js

```diff
@@ -40,9 +40,7 @@ if (!Math.clz32) Math.clz32 = function(x) {
 
 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/trunc
 #if POLYFILL_OLD_MATH_FUNCTIONS || MIN_CHROME_VERSION < 38 || MIN_EDGE_VERSION < 12 || MIN_FIREFOX_VERSION < 25 || MIN_IE_VERSION != TARGET_NOT_SUPPORTED || MIN_SAFARI_VERSION < 80000 // || MIN_NODE_VERSION < 0.12
-if (!Math.trunc) Math.trunc = function(x) {
-  return x < 0 ? Math.ceil(x) : Math.floor(x);
-};
+if (!Math.trunc) Math.trunc = (x) => x < 0 ? Math.ceil(x) : Math.floor(x);
```

Comment:
The code in this file is for emulating old browsers - if they don't support some of these, they will not support ES6 either.